phonebook.py

```diff
@@ -2,7 +2,34 @@
 class Phonebook:
 
     def __init__(self):
-        pass
+        self.phonebook = {}
 
-    def add_contacts():
-        pass
+    def add_contact(self, name, contact):
+
+        if contact.isdigit() and (name != "" or name != " "):
+            self.phonebook[name] = contact
```

Comment:
What do you think about trying to check if the contact was added to the phonebook dict and return a successful message?